src/main/java/edu/ie3/datamodel/io/source/csv/CsvDataSource.java

```diff
@@ -44,7 +50,7 @@ public class CsvDataSource implements DataSource {
 
   public CsvDataSource(String csvSep, Path folderPath, FileNamingStrategy fileNamingStrategy) {
     this.csvSep = csvSep;
-    this.connector = new CsvFileConnector(folderPath, fileNamingStrategy);
+    this.connector = new CsvFileConnector(folderPath);
```

Comment:
Please rename `folderPath` -> `directoryPath`, also in all other files that you touched here